maestro-client/src/main/java/maestro/Driver.kt

```diff
@@ -84,5 +84,15 @@ interface Driver {
 
     fun isUnicodeInputSupported(): Boolean
 
-    fun isScreenStatic(): Boolean
+    fun waitUntilScreenIsStatic(timeoutMs: Long): Boolean
+
+    fun waitForAppToSettle(initialHierarchy: ViewHierarchy?): ViewHierarchy?
+
+    fun performTap(
```

Comment:
Let's not include it into a `Driver`, it should not be concerned with that higher-level logic